docker-compose.dev.yml

```diff
@@ -0,0 +1,21 @@
+services:
+  app:
+    build:
+      context: .
+      target: dev
+    container_name: app-api
```

Comment:
No need

Reply:
It is used in Makefile cmd.

Reply:
Anyway, it raises conflicts if you want to run two different project